recipe/meta.yaml

```diff
@@ -0,0 +1,161 @@
+
+build:
+  number: 0
+  skip: True  # [py>311]
+  # temporarily skip windows until ffmpeg is available
+  script:
+    - export USE_FFMPEG=ON          # [not win]
+    - set USE_FFMPEG=ON             # [win]
+    - export BUILD_SOX=OFF          # [not win]
+    - set BUILD_SOX=OFF             # [win]
+    - export FFMPEG_ROOT=${PREFIX}  # [not win]
+    - set FFMPEG_ROOT=${PREFIX}     # [win]
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+    - install_name_tool -add_rpath @loader_path/ $SP_DIR/torchaudio/lib/_torchaudio_ffmpeg.so  # [osx]
+    - install_name_tool -add_rpath @loader_path/ $SP_DIR/torchaudio/lib/_torchaudio.so         # [osx]
+    - patchelf --set-rpath '$ORIGIN' $SP_DIR/torchaudio/lib/_torchaudio_ffmpeg.so                  # [linux]
+    - patchelf --add-rpath '$ORIGIN/../../../..' $SP_DIR/torchaudio/lib/_torchaudio_ffmpeg.so      # [linux]
+    - patchelf --add-rpath '$ORIGIN/../../torch/lib' $SP_DIR/torchaudio/lib/_torchaudio_ffmpeg.so  # [linux]
+    - patchelf --set-rpath '$ORIGIN/../../torch/lib' $SP_DIR/torchaudio/lib/_torchaudio.so         # [linux]
+    - patchelf --add-rpath '$ORIGIN' $SP_DIR/torchaudio/lib/_torchaudio.so                         # [linux]
+  ignore_run_exports:
+    - pytorch-gpu  # [osx and arm64]
+    - libprotobuf  # [osx and arm64]
+    - libabseil    # [osx and arm64]
+  missing_dso_whitelist:
+    - lib/libstdc++.so.*            # [linux]
+    - lib/libgcc_s.so.*             # [linux]
+    - lib/libgomp.so.*              # [linux]
+    - $RPATH/ld64.so.1              # [linux and s390x]
+    - x86_64-conda-linux-gnu/sysroot/lib64/libm.so.*        # [linux and x64_64]
+    - x86_64-conda-linux-gnu/sysroot/lib64/libc.so.*        # [linux and x64_64]
+    - x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0  # [linux and x64_64]
+    - s390x-conda-linux-gnu/sysroot/lib64/libc.so.*         # [linux and s390x]
+    # contained in ffmpeg
+    - $RPATH/libavutil.*.dylib      # [osx]
+    - $RPATH/libavcodec.*.dylib     # [osx]
+    - $RPATH/libavformat.*.dylib    # [osx]
+    - $RPATH/libavdevice.*.dylib    # [osx]
+    - $RPATH/libavfilter.*.dylib    # [osx]
+    - $RPATH/libavutil.so.*         # [linux]
+    - $RPATH/libc10.so              # [linux]
+    - $RPATH/libavutil.so.*         # [linux]
+    - $RPATH/libavformat.so.*       # [linux]
+    - $RPATH/libavfilter.so.*       # [linux]
+    # libsox shouldn't be activated to begin with
+    - $RPATH/libsox.dylib           # [osx]
+    # in pytorch package
+    - $RPATH/libtorch.dylib         # [osx]
+    - $RPATH/libtorch_cpu.dylib     # [osx]
+    - $RPATH/libtorch_python.dylib  # [osx]
+    - $RPATH/libtorch_cpu.so        # [linux]
+    - $RPATH/libtorch_python.so     # [linux]
+    - $RPATH/libc10.dylib           # [osx]
+    # in protobuf package
+    - $RPATH/libprotobuf.*.dylib    # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - make  # [not win]
```

Comment:
ninja would be better

Reply:
do we replace `make` with `ninja` by default?

Reply:
You can export `CMAKE_GENERATOR` in this case and cmake will pick it up.

Reply:
but I still have to include `ninja`, right? in this case I would just replace it if there's really no other modification to make.

Reply:
Yep, you will have to replace `make` with `ninja`.